mappings/net/minecraft/client/gui/EditBox.mapping

```diff
@@ -14,8 +14,8 @@ CLASS net/minecraft/class_7530 net/minecraft/client/gui/EditBox
 	FIELD field_39518 selecting Z
 	FIELD field_39519 maxLength I
 	FIELD field_39520 width I
-	FIELD field_39521 changeCallback Ljava/util/function/Consumer;
-	FIELD field_39522 cursorChangeCallback Ljava/lang/Runnable;
+	FIELD field_39521 changeListener Ljava/util/function/Consumer;
+	FIELD field_39522 cursorChangeListener Ljava/lang/Runnable;
```

Comment:
For bikeshedding, I'd just call them `textListener` and `cursorListener` as listeners/callbacks really only respond to (implies) changes